pkg/devspace/configure/image.go

```diff
@@ -174,7 +174,7 @@ func (m *manager) AddImage(imageName, image, projectNamespace, dockerfile string
 				registryHostname, err = m.log.Question(&survey.QuestionOptions{
 					Question:               "Please provide the registry hostname without the image path (e.g. gcr.io, ghcr.io, ecr.io)",
 					DefaultValue:           "gcr.io",
-					ValidationRegexPattern: "^(?!-)[a-z0-9-]{1,63}(\\.[a-z0-9-]{1,63})*$",
+					ValidationRegexPattern: "^[a-z0-9][a-z0-9-]{0,62}(\\.[a-z0-9-]{1,63})*$",
```

Comment:
only nit: use backtick string to avoid the double escape `\\.`:
```suggestion
					ValidationRegexPattern: `^[a-z0-9][a-z0-9-]{0,62}(\.[a-z0-9-]{1,63})*$`,
```